File: code-samples/python/feeds@products@price.py
import requests

url = "https://vendor-api-staging.jumia.com/feeds/products/price"

payload = {"products": [
        {
            "id": "string",
            "sellerSku": "string",
            "category": 0,
            "price": {
                "currency": "string",
                "value": 0,
                "salePrice": {
                    "value": 0,
                    "startAt": "string",
                    "endAt": "string"
                }
            },
            "businessClients": [
                {
                    "businessClientCode": "string",
                    "price": {
                        "currency": "string",
                        "value": 0,
                        "salePrice": {
                            "value": 0,
                            "startAt": "string",
                            "endAt": "string"
                        }
                    }
                }
            ]
        }
    ]}
headers = {"Authorization": "Bearer "}

response = requests.request("POST", url, json=payload, headers=headers)

print(response.text)

File: code-samples/go/feeds@products@price.go
package main

import (
  "fmt"
  "strings"
  "net/http"
  "io/ioutil"
)

func main() {

  url := "https://vendor-api-staging.jumia.com/feeds/products/price"

  payload := strings.NewReader("{\n  \"products\": [\n    {\n      \"id\": \"string\",\n      \"sellerSku\": \"string\",\n      \"category\": 0,\n      \"price\": {\n        \"currency\": \"string\",\n        \"value\": 0,\n        \"salePrice\": {\n          \"value\": 0,\n          \"startAt\": \"string\",\n          \"endAt\": \"string\"\n        }\n      },\n      \"businessClients\": [\n        {\n          \"businessClientCode\": \"string\",\n          \"price\": {\n            \"currency\": \"string\",\n            \"value\": 0,\n            \"salePrice\": {\n              \"value\": 0,\n              \"startAt\": \"string\",\n              \"endAt\": \"string\"\n            }\n          }\n        }\n      ]\n    }\n  ]\n}")

  req, _ := http.NewRequest("POST", url, payload)

  req.Header.Add("Authorization", "Bearer ")

  res, _ := http.DefaultClient.Do(req)

  defer res.Body.Close()
  body, _ := ioutil.ReadAll(res.Body)

  fmt.Println(res)
  fmt.Println(string(body))

}